source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationsAttribute.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using Octopus.Tentacle.Util;
+
+namespace Octopus.Tentacle.Tests.Integration.Support
+{
+    public class TentacleConfigurationsAttribute : TestCaseSourceAttribute
+    {
+        public TentacleConfigurationsAttribute(
+            bool testCommonVersions = false,
+            bool testCapabilitiesServiceInterestingVersions = false,
+            bool testNoCapabilitiesServiceVersions = false,
+            bool testScriptIsolationLevelVersions = false,
+            params object[] additionalParameterTypes)
+            : base(
+                typeof(TentacleConfigurationTestCases),
+                nameof(TentacleConfigurationTestCases.GetEnumerator),
+                new object[] {testCommonVersions, testCapabilitiesServiceInterestingVersions, testNoCapabilitiesServiceVersions, testScriptIsolationLevelVersions, additionalParameterTypes})
+        {
+        }
+    }
+
+    static class TentacleConfigurationTestCases
+    {
+        public static IEnumerator GetEnumerator(
+            bool testCommonVersions,
+            bool testCapabilitiesInterestingVersions,
+            bool testNoCapabilitiesServiceVersions,
+            bool testScriptIsolationLevel,
```

Comment:
Nit: `testScriptIsolationLevel` different name to `testScriptIsolationLevelVersions` above